Cargo.toml

```diff
@@ -35,6 +35,10 @@ dbg = []
 cert-logging = ["base64"]
 # Used for nicely documenting the Android-specific APIs. This feature is not stable.
 docsrs = ["jni", "once_cell"]
+# Enables API for Linux verifier to be provided extra non-platform-provided roots (e.g.
+# from `webpki-roots`). This can be helpful to augment the platform-provided roots
+# when the platform-provided roots are out of date.
+extra-roots = []
```

Comment:
This seems like a small, portable API. Why bother masking it behind a feature?

Reply:
I understand djc to be thinking of a webpki-specific API, but I think the less opinionated approach you have here is preferable in part *because* there's no need to gate it.

Reply:
I like the idea of always exposing it. That would also match up better with [the API in `rustls` itself](https://docs.rs/rustls/latest/rustls/struct.ConfigBuilder.html#method.with_root_certificates) which is always available even without the dangerous configuration feature enabled.

Reply:
I generally wouldn't mask extra API behind a feature unless it requires extra dependencies or there is another motivation (like for dangerous configuration). I agree with @Ralith that it probably doesn't make sense here.

Reply:
GitHub put my comment elsewhere by mistake, but I'm also in favor of this change. It also lines up with the `rustls` API for adding arbitrary roots to a config not being gated.

Reply:
Removed the feature flag. Thanks for the input all :+1: